.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: python
+# cache package wheels (1 cache per python version)
+cache: pip
+# newer python versions are available only on xenial (while some older only on trusty) Ubuntu distribution
+dist: xenial
+sudo: required
+
+env:
+  TOXENV=py
+
+jobs:
+  include:
+  - name: "run test suite with python 3.4"
+    python: 3.4
+    dist: trusty
+  - name: "run test suite with python 3.5"
+    python: 3.5.1
+    dist: trusty
+  - name: "run test suite with python 3.6"
+    python: 3.6
+  - name: "run test suite with python 3.7"
+    python: 3.7
+  - name: "run test suite with python 3.8-dev"
+    python: 3.8-dev
+  - name: "check code style with flake8"
+    python: 3.7
+    env:
+    - TOXENV=lint
+
+install:
+- pip install -U pip setuptools
+- pip install -U tox
+- tox --notest
+
+script:
+- tox
```

Comment:
Since this is a "runtime" package, I would rather use the Travis settings from `python/typing`.

Reply:
Which things from typing do you prefer? Is it just running on a bunch more versions, or do you want to not use tox?

Reply:
Actually both (but first is probably more important).

Reply:
What's the reason to avoid tox? (I don't have any real opinion one way or another except that the tox version works already :P)

Reply:
I just don't want to learn something the need for what I don't understand. But since it already works, then OK.